src/styles/_catalog.scss

```diff
@@ -13,10 +16,30 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    box-sizing: border-box;
+  }
+}
+
+@media screen and (min-width: 320px) {
+  .catalog {
+    grid-template-columns: $card-width;
   }
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
```

Comment:
You can move all media requests inside the catalog to avoid duplicating the class name so many times, but this is optional